### PR TITLE
[Keymap] kifinnsson's dz60 keymap

### PR DESCRIPTION
**Pull request by kifinnsson**
## kifinnsson's keymap for the DZ60 Keyboard
Keymap for my non-standard DZ60 layout. It is an ansi layout on the right and iso on the left (ie 1.25x left shift). This is to implement the angle mod on for Colemak which is the base layer. A side effect of this is that I have an extra key on row 4, which sits between the "b" and "k" keys in Colemak. I use this key as a switch to layer 2 which is my macro layer.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by kifinnsson** [comment]
My fork was pretty messy, I have reforked it and cleaned things up as suggested, Thanks for the suggestions, hopefully it's in better shape this time.

File: keyboards/dz60/keymaps/kifinnsson/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: DZ60
+Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/collections/pcb/products/dz60-60-pcb?variant=40971616717)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make dz60:kifinnson
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/kifinnsson/readme.md]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/dz60/keymaps/kifinnsson/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+kifinnsson's Colemak angle mod ansi layout
+-----------------
+This is built as an ISO layout on the left and an ANSI layout on the right. This creates an [Angle Mod](https://colemakmods.github.io/ergonomic-mods/angle.html) ANSI Colemak layout however row 4 has an extra key which I place between "B" and "K" and is used for my macros on layer 2. If you build one with my wonkyness feel free to use it as a template. If you didn't build as I did you are not going to have a good time. This a cleaned up / sanitized version for pubilshing upstream. 
+
+![dz60](https://cdn.shopify.com/s/files/1/1473/3902/files/1_6525343b-ee62-47e8-882a-05e316136a3f.jpg?v=1501657073)
+
+A customizable 60% keyboard.
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: DZ60
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/kifinnsson/readme.md]:
```suggestion
Hardware Supported: DZ60  
```

**Comment by kifinnsson** [comment]
my readme.md was out of date, it was still using the default for the keyboard. I missed that in my cleanup I'll open a new pull later.

**Comment by fauxpark** [comment]
You can leave this PR open and add it here, no need to make a new PR.

**Pull request closed without merging**